build.go

```diff
@@ -149,7 +155,11 @@ func build(folder string) error {
 		}
 		return runCmds(folder, env, cmds)
 	case "xz":
-		return runCmds(folder, nil, [][]string{
+		var env []string
+		if runtime.GOOS == "darwin" {
+			env = []string{"PATH=" + autopointPath + ":" + os.Getenv("PATH")}
+		}
```

Comment:
Mentioned above, but make the caller do this

Reply:
After offline discussion, going to leave this to keep the user from having to litter their PATH